Directory.Packages.props

```diff
@@ -79,47 +79,47 @@
+    <PackageVersion Include="KubernetesClient" Version="16.0.2" />
+    <PackageVersion Include="JsonPatch.Net" Version="3.3.0" />
     <PackageVersion Include="Microsoft.Data.SqlClient" Version="6.0.1" />
     <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.11.3" />
     <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components.Icons" Version="4.11.3" />
     <PackageVersion Include="Milvus.Client" Version="2.3.0-preview.1" />
     <PackageVersion Include="MongoDB.Driver" Version="3.1.0" />
     <PackageVersion Include="MongoDB.Driver.Core.Extensions.DiagnosticSources" Version="2.0.0" />
-    <PackageVersion Include="MySqlConnector.DependencyInjection" Version="2.3.6" />
+    <PackageVersion Include="MySqlConnector.DependencyInjection" Version="2.4.0" />
     <PackageVersion Include="MySqlConnector.Logging.Microsoft.Extensions.Logging" Version="2.1.0" />
-    <PackageVersion Include="NATS.Net" Version="2.5.3" />
+    <PackageVersion Include="NATS.Net" Version="2.5.7" />
     <PackageVersion Include="Npgsql.DependencyInjection" Version="9.0.2" />
     <PackageVersion Include="OpenAI" Version="2.1.0" />
-    <PackageVersion Include="Oracle.EntityFrameworkCore" Version="8.23.60" />
-    <PackageVersion Include="Oracle.ManagedDataAccess.OpenTelemetry" Version="23.6.0" />
+    <PackageVersion Include="Oracle.EntityFrameworkCore" Version="9.23.60" />
+    <PackageVersion Include="Oracle.ManagedDataAccess.OpenTelemetry" Version="23.7.0" />
     <PackageVersion Include="Polly.Core" Version="8.5.1" />
     <PackageVersion Include="Polly.Extensions" Version="8.5.1" />
     <PackageVersion Include="Pomelo.EntityFrameworkCore.MySql" Version="8.0.2" />
-    <PackageVersion Include="Qdrant.Client" Version="1.12.0" />
+    <PackageVersion Include="Qdrant.Client" Version="1.13.0" />
     <PackageVersion Include="RabbitMQ.Client" Version="7.0.0" />
-    <PackageVersion Include="StackExchange.Redis" Version="2.8.22" />
+    <PackageVersion Include="StackExchange.Redis" Version="2.8.24" />
     <PackageVersion Include="System.IO.Hashing" Version="9.0.1" />
     <PackageVersion Include="Yarp.ReverseProxy" Version="2.2.0" />
     <!-- Open Telemetry -->
     <PackageVersion Include="Npgsql.OpenTelemetry" Version="9.0.2" />
-    <PackageVersion Include="OpenTelemetry.Exporter.InMemory" Version="1.9.0" />
-    <PackageVersion Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.9.0" />
-    <PackageVersion Include="OpenTelemetry.Extensions.Hosting" Version="1.9.0" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.GrpcNetClient" Version="1.9.0-beta.1" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.Http" Version="1.9.0" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.9.0" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.Runtime" Version="1.9.0" />
+    <PackageVersion Include="OpenTelemetry.Exporter.InMemory" Version="1.11.1" />
```

Comment:
We don't want to update OpenTelemetry right now. [.NET Aspire is stuck on OpenTelemetry 1.9.0 (dotnet/aspire#7123)](https://github.com/dotnet/aspire/issues/7123)

Reply:
yeah, for all of these comments, I think that if we are to merge this workflow into the main fork of aspire, we'd want to use it as a "notification" for all newer versions, but then we'll want to push changes to it to undo the ones we don't want/need